src/commands/deployWorkspaceProject/deployWorkspaceProject.ts

```diff
@@ -37,8 +39,13 @@ export async function deployWorkspaceProject(context: IActionContext & Partial<D
         subscription
     });
 
+    ext.outputChannel.appendLog(
+        formatSectionHeader(localize('preparingDeploymentConfiguration', 'Preparing workspace deployment configuration'))
```

Comment:
I would say anything that sounds like a long running action should end with a `...`

Verbs ending with "ing" tend to make me thing, on-going long running action.

Reply:
I'll make it just "Prepare" then.  I'm trying to make it look more like a title/header than an active loading command.